CHANGELOG.md

```diff
@@ -20,11 +20,10 @@
 All notable changes to this project will be documented in this file.
 
 ## 0.9.1
-
 * ### Merged Pull Requests
     *  ...
 * ### Major Changes
-    * ...
+    * Fix naming issues. `AAND_B` is now `A_AND_B`,  `R_1` is now `R1`,  `R_2` is now `R2`,  `R_3` is now `R3` and  `R_4` is now `R4`.
```

Comment:
```suggestion
    * Add support for strain calibration (offset nulling and shunt calibration)
    * Fix naming issues:
      * Rename `ShuntCalSelect.AAND_B` to `A_AND_B`.
```

No need to mention R_1 etc. because they didn't exist before this release.

FYI @DeborahOoi96 one bullet can cover both strain cal features